integration_tests/test_redis_myco_matcher.py

```diff
@@ -23,7 +25,7 @@ def on_tick(self, data):
 
     def on_offers_bids_response(self, data):
         self.called_events.add("offers_bids_response")
-        recommendations = PayAsBidMatchingAlgorithm.get_matches_recommendations(
+        recommendations = AttributedMatchingAlgorithm.get_matches_recommendations(
```

Comment:
Doesn't this mean that you're deleting all tests for the `PayAsBidMatchingAlgorithm`?